src/Container/Container.php

```diff
@@ -17,9 +19,14 @@ public function config(object $config): self;
      * @param class-string<TClassName> $className
      * @return TClassName
      */
-    public function get(string $className): object;
+    public function get(string $className, mixed ...$params): object;
```

Comment:
I added support for passing extra params in `get`. I did this when I was debugging some issues during implementation with circular dependencies.

In the end, it turns out I didn't need it anyway. So I might remove it still. On the other hand, it might be a nice feature, and we already support it via `call`. So why not include it here?

Reply:
I think we should totally leave this here. It'd be required anytime you need primitives added anyway.